Assets/Scenes/Levels/Level_1.unity

```diff
@@ -530,6 +535,134 @@ Transform:
   m_CorrespondingSourceObject: {fileID: 7936487194590973691, guid: ac3f13da9a926624c9281b854d1c7caf, type: 3}
   m_PrefabInstance: {fileID: 976261184}
   m_PrefabAsset: {fileID: 0}
+--- !u!1 &1620749469
```

Comment:
Looks like you may have added a game object with an audio source only to Level 1. Maybe you modified the prefab and need to override it so it goes to other scenes?